Windows/debugger.cpp

```diff
@@ -1536,8 +1536,35 @@ DebuggerStatus Debugger::Attach(unsigned int pid, uint32_t timeout) {
   attach_mode = true;
 
   if (!DebugActiveProcess(pid)) {
-    FATAL("Could not attach to the process.\n"
-          "Make sure the process exists and you have permissions to debug it.\n");
+    DWORD error_code = GetLastError();
+    SAY("DebugActiveProcess(%d) failed with error code = %d\n", error_code);
+
+    if(error_code == 5) {

```

Comment:
This 'if' is missing an 'else', that is, `DebugActiveProcess` could fail for some other reason.

Reply:
I edit code style as you suggested ([commit link](https://github.com/googleprojectzero/TinyInst/pull/35/commits/23f0b23cfd612b92245bdbf1e1f36dc34d44aeb1))